packages/amplify-category-auth/package.json

```diff
@@ -41,6 +41,7 @@
     "aws-sdk": "^2.1169.0",
     "chalk": "^4.1.1",
     "change-case": "^4.1.1",
+    "cloudform-types": "^4.2.0",
```

Comment:
This is already listed below in `devDependencies`.

Perhaps changing the two times it's imported from 

```ts
import { Template } from 'cloudform-types';
```

to 

```ts
import type { Template } from 'cloudform-types';
```

would resolve the need to add this to `dependencies`.